src/v2/Apps/Artist/Utils/computeTitle.ts

```diff
@@ -1,13 +1,16 @@
 /**
- * Computes title meta tage value for an artist
+ * Computes title meta tag value for an artist
  */
-export function computeTitle(name: string, count: number) {
-  const parts = [
-    name,
-    "-",
-    count ? count + " Artworks," : null,
-    "Bio & Shows on Artsy",
-  ]
-  const titleString = parts.filter(Boolean).join(" ")
-  return titleString
+export function computeTitle(
+  name: string,
+  count: number,
+  forSaleOnly: boolean = false
+) {
+  if (count > 0) {
+    return `${name} - ${count} Artworks${
```

Comment:
I realize this logic was already here as is but:

* The count might be 1 so we'd want to conditionally add the s to artworks
* We might not have a count at all

Reply:
(Oh I see you accommodate the count not existing)

Reply:
In the case that count is undefined, it should evaluate as false, right?

Reply:
```js
undefined > 0
// false
```

👍